### PR TITLE
[i18n] Update and extract all user-facing strings

### PR DESCRIPTION
**Pull request by Tobotimus**
### Aim
This PR aims to raise the standard to improve the coverage of extracted user-facing string literals and to make them less ambiguous for the translator, consequently improving the quality of translations.

### Rationale
A lot of our user-facing strings are either not being passed to gettext (they need the `_()` around them) or they're formatted strings being passed with ambiguous placeholders (see #1526).

#### Why a feature branch?
I've created a feature branch here to allow more people to contribute to the effort. It's quite a lot of work for just one person to do alone. FYI PR's you make to this branch will be squashed, however when this PR gets merged it will retain all individual commits, so you will get attributed for work you've done.

@mikeshardmind started us off with his work in #1795, so I've included those additions in here :)

#### Notes for contributing
Do not use f-strings when putting a string in gettext, they *must* be in the following format:
```py
_("Ping {time}ms").format(time=random.randint())
```

Notice the call to `.format()` being *outside* the call to `_()`.

This **will not** work either, as gettext will not find the string in question:
```py
f"{_('Ping {time}ms')}"
```

#### Cog packages
- [x] admin
- [x] alias
- [x] audio
- [x] bank
- [x] cleanup
- [x] customcom
- [x] dataconverter
- [x] downloader
- [x] economy
- [x] filter
- [x] general
- [x] image
- [x] mod
- [x] modlog
- [x] permissions
- [x] reports
- [x] streams
- [x] trivia
- [x] warnings

#### Core modules

- [ ] commands (including discord.py converters)
- [ ] utils
- [ ] bank
- [ ] cog_manager
- [ ] core_commands
- [ ] dev_commands
- [ ] events
- [ ] help_formatter
- [ ] modlog

### CODE REVIEW

File: redbot/cogs/trivia/trivia.py [hunk 1]
```diff
@@ -437,7 +437,7 @@ def _get_leaderboard(self, data: dict, key: str, top: int):
 
         """
         channel = session.ctx.channel
-        LOG.debug("Ending trivia session; #%s in %s", channel, channel.guild.id)
+        LOG.debug(f"Ending trivia session; #{channel} in {channel.guild}")
```
Comment (mikeshardmind) [inline redbot/cogs/trivia/trivia.py]: This (and any other log statements) should be reverted back to using % formatting. 

Unlike f-string usage, formatting is delayed for evaluation until the result is needed, and won't ever get formatted if the log level doesn't demand it.

**Comment by Tobotimus** [comment]
I've just done a force push because the commit history was really whacky before and was causing issues. If you have this feature branch checked out locally, stash whatever changes you've made, delete your local branch, pull from upstream and re-apply. Sorry for any inconvenience.

**Comment by Tobotimus** [comment]
I'm just about to rebase and force-push this PR's branch

**Comment by Tobotimus** [comment]
I'm going to merge this PR as it currently stands before RC1. At this point, the `redbot.cogs` package has been updated to a good standard. I will try to get the `redbot.core` package done for RC2.

**Pull request merged**